index.js

```diff
@@ -0,0 +1,98 @@
+let myLibrary = [];
+const auth = document.getElementById('author');
+const title = document.getElementById('title');
+const pages = document.getElementById('pages');
+const read = document.getElementById('read');
+const btn = document.getElementById('submit');
+const table = document.getElementById('table');
+
+function getBooks() {
+  let data = localStorage.getItem('library');
+  data = JSON.parse(data);
+  return data;
+}
+function setBooks() {
+  const data = JSON.stringify(myLibrary);
+  localStorage.setItem('library', data);
+}
+
+function Book(author, title, pages, read = false) {
+  this.author = author;
+  this.title = title;
+  this.pages = pages;
+  this.read = read;
+}
+
+function addBookToLibrary(arr, book) {
+  arr.push(book);
+  setBooks();
+  window.location.reload();
+}
+
+
+const getter = getBooks();
+myLibrary = [...getter];
+getBooks();
+if (getter.length === 0) {
+  getter.push(new Book('Neko Master', '9999 Reasons I love my mommy', '9999999', true));
+  myLibrary = [...getter];
+  setBooks();
+}
+
+btn.addEventListener('click', (e) => {
+  e.preventDefault();
+  if (auth.value !== '' && title.value !== '' && pages.value !== '') {
+    const book = new Book(auth.value, title.value, pages.value, read.checked);
+    addBookToLibrary(myLibrary, book);
+  }
+});
+
+function displayEachBook() {
```

Comment:
## Implement function with ES6 arrow function
- [ ] Implement this `displayEachBook` function with ES6 arrow function
```suggestion
const displayEachBook = () => {
```